src/main/java/com/crowdin/cli/commands/picocli/StringListSubcommand.java

```diff
@@ -32,16 +35,26 @@ class StringListSubcommand extends ActCommandProject {
     @CommandLine.Option(names = {"--croql"}, paramLabel = "...", order = -2)
     protected String croql;
 
+    @CommandLine.Option(names = {"--directory"}, paramLabel = "...", order = -2)
+    protected Long directory;
```

Comment:
Since we accept the file path in the `--file` option, it would be better to use the same approach here by accepting the directory path. Also, we can't get the directory ID with CLI